receipt_dynamo/tests/unit/test_receipt_summary_printed_total.py

```
@@ -0,0 +1,228 @@
+        _word(2, 1, "42.54", 0.55),
+    ]
+    assert find_printed_grand_total(words) is None
+
+
+def test_summary_fallback_recovers_total_from_rejected_labels():
+    # Real failure mode: every GRAND_TOTAL label sits on a garbage word
+    # ("Cashier:", "@") and is INVALID, so the label pass yields None.
+    words = _sprouts_a0301717_words() + [
+        _word(71, 1, "Cashier:", 0.8182, x=0.803),
+        _word(37, 1, "@", 0.4071, x=0.808),
+    ]
```

Ruff (0.16.0)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)